util/chplenv/chpl_launcher.py

```diff
@@ -44,6 +44,8 @@ def get():
                 launcher_val = 'gasnetrun_mpi'
             elif substrate_val == 'ibv':
                 launcher_val = 'gasnetrun_ibv'
+            elif substrate_val == 'ucx':
+                launcher_val = 'gasnetrun_ibv'
```

Comment:
gasnetrun_ucx?

Reply:
Whoops, yeah good catch. Fixed.

Reply:
(In my testing I'm manually using `slurm-gasnetrun_ucx` -- and I opened https://github.com/Cray/chapel-private/issues/1729 to auto-detect slurm)